src/base/static/js/views/app-view.js

```diff
@@ -340,28 +338,13 @@ export default Backbone.View.extend({
     });
     // END REACT PORT SECTION //////////////////////////////////////////////////
 
-    // When the map center moves, the map view will fire a mapmoveend event
```

Comment:
I don't think this reverse geocoding code has done anything for a while, so I removed it. Eventually though we should restore reverse geocoding.